.github/workflows/package.yml

```diff
@@ -0,0 +1,72 @@
+name: Test and build Python package
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - v*
```

Comment:
Tags for this repo have not historically used the `v` prefix. Should we adopt this practice moving forward, or update this regex? Cc: @fgregg

Reply:
sure, let's use tags with v